src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -292,24 +292,6 @@ public bool ShowCodeBlockLineNumber
             }
         }
 
-        /// <summary>
-        /// Indicates whether to make T-Spline nodes (under ProtoGeometry.dll) discoverable
-        /// in the node search library.
-        /// </summary>
-        public bool EnableTSpline
```

Comment:
Unfortunately we can only Obsolete this

Reply:
Just added this methods back and added the obsolete tag, I had to also bring back `internal void HideUnhideNamespace(bool hide, string library, string namespc)` in DynamoModel.cs as it is used by one of them